homeassistant/helpers/device_registry.py

```diff
@@ -20,6 +20,7 @@
 
 CONNECTION_NETWORK_MAC = 'mac'
 CONNECTION_ZIGBEE = 'zigbee'
+CONNECTION_KEYS = [CONNECTION_NETWORK_MAC, CONNECTION_ZIGBEE]
```

Comment:
One problem with this will be that every time a new connection "type" is added, the MQTT docs would need to be updated as well. Should the connection `type` option therefore not be validated?

Reply:
Don't validate it. Those are suggested entries, not all possible entries.